src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  & :not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--active :not(:last-child) {
+    background-image: url(/src/images/star-active.svg);
+  }
+
+  &__item {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
```

Comment:
Ensure that the image `star.svg` exists at the path `/src/images/`. If it's not present, the star background will not display correctly.